src/components/SubRequirement.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <div class="subrequirement">
+  <div class="row depth-req">
+    <div class="col-1" @click="toggleDescription(index, isCompleted, id)">
+      <button class="btn">
+        <!-- svg for dropdown icon -->
+        <img
+          v-if="subReq.displayDescription && !isCompleted"
+          class="arrow arrow-up"
+          src="@/assets/images/dropup.svg"
+          alt="dropup"
+        />
+        <img
+          v-if="subReq.displayDescription && isCompleted"
+          class="arrow arrow-up"
+          src="@/assets/images/dropup-lightgray.svg"
+          alt="dropup"
+        />
+        <img
+          v-if="!subReq.displayDescription && !isCompleted"
+          class="arrow arrow-down"
+          src="@/assets/images/dropdown.svg"
+          alt="dropdown"
+        />
+        <img
+          v-if="!subReq.displayDescription && isCompleted"
+          class="arrow arrow-down"
+          src="@/assets/images/dropdown-lightgray.svg"
+          alt="dropdown"
+        />
```

Comment:
I was trying to find a way to conditionally change src value like how I did for class but I didn't have any luck. If anyone has suggestions to improve this, please lmk!

Reply:
The best thing I can think of right now is calling a function in src that returns the correct string. So defining something like dropImage() that returns the up/down/gray variants depending on those two bools. If you do that you would probably only need one <img> tag so it might be better? Not sure though.

Reply:
@willespencer Yeah I tried to define a function that did this but for some reason the value was a str and not the actual relative path if that makes sense. So the string itself would correctly be "@/assets/images/dropdown-lightgray.svg" but it was as if I didn't have a relative path at all. It would as if I typed in any other string like "hi". Would it be ok to keep it like this for now or should I try to find another way to make the code cleaner?

Reply:
Resolved over slack!